src/main/scala/com/github/unchama/seichiassist/subsystems/tradesystems/subsystems/gachatrade/bukkit/listeners/GachaTradeListener.scala

```diff
@@ -74,10 +74,11 @@ class GachaTradeListener[F[_]: ConcurrentEffect, G[_]: ContextCoercion[*[_], F]]
      */
     val tradableItemStacks = tradedInformation.tradedSuccessResult
     val bigItemStackAmounts = tradableItemStacks.collect {
-      case result if result.transactionInfo._1 == BigOrRegular.Big => result.amount
+      case result if result.transactionInfo._1 == BigOrRegular.Big => result.transactionInfo._2
```

Comment:
`import com.github.unchama.seichiassist.subsystems.tradesystems.domain.TradeSuccessResult`
した上で
```suggestion
      case TradeSuccessResult(_, _, (rarity, amount)) if rarity == BigOrRegular.Big => amount
```
とすることができます。